packages/medusa/src/api/routes/admin/discounts/create-discount.ts

```diff
@@ -83,8 +83,6 @@ import { AdminPostDiscountsDiscountParams } from "./update-discount"
 export default async (req, res) => {
```

Comment:
this file is also added in [that PR](https://github.com/medusajs/medusa/pull/1355/files#diff-c660cf34811108809d8751be579710b20bf10ec78068da7f90c39bf469875d7b) is that normal

Did you create that branch from oli PR?

Reply:
Yes, I branched from Oli's branch to get the proper request classes.